qc2md.py

```diff
@@ -158,12 +170,16 @@ def parse_report(lines: list[str]) -> list[QCEntry]:
 
 
 def categorize_entries(
-    entries: list[QCEntry], *, group_script_entries: bool = False
+    entries: list[QCEntry],
+    standalone_categories: list[str],
```

Comment:
Am unsure about making this a mandatory argument, as it has no effect unless `group_script_entries`—an optional, and keyword-only argument—is enabled.